specs/namespaces/namespaces_spec.py

```diff
@@ -306,3 +310,12 @@
                 {'data': {'type': 'flight-offers-pricing',
                           'flightOffers': [{'foo': 'bar'}]}}
             ))
+        with it('.shopping.booking.flight_orders.post'):
+            self.client.booking.flight_orders.post({'foo': 'bar'}, {'bar': 'foo'})
```

Comment:
do we miss the unit test for the flight order management get?

Reply:
The unit test is here https://github.com/amadeus4dev/amadeus-python/pull/62/files/ab18882346a41e2468aa7854316f8a3239c9cf80#diff-6c2042d33b9a79ce08554c558ee65e69R117

Reply:
ah right, there is one more missing. Adding it now